drools-wb-screens/drools-wb-guided-dtable-editor/drools-wb-guided-dtable-editor-client/src/main/java/org/drools/workbench/screens/guided/dtable/client/widget/table/GuidedDecisionTableModellerViewImpl.java

```diff
@@ -244,6 +249,18 @@ public HandlerRegistration addKeyDownHandler(final KeyDownHandler handler) {
         return gridPanel.addKeyDownHandler(handler);
     }
 
+    void enableButtonMenu() {
+        getSubMenu().setVisible(true);
```

Comment:
Enabling/disabling method names; however you're changing the visibility. It'd be nice to have the method names and implementation be consistent (personally I prefer to disable the buttons rather than hide as the User can still _see_ there's an action they can perform but it's disabled).